fuzz-proto/fuzz-v1.asn

```diff
@@ -0,0 +1,96 @@
+-- Specification for Fuzzer Protocol V1
+
+FuzzProtocol DEFINITIONS ::= BEGIN
+
+-- `Block`, `Header`, `WorkReport`, `SegmentRootLookup` as defined by gray paper. 
+-- See also https://github.com/w3f/jamtestvectors/blob/master/lib/jam-types.asn
+
+-- Refer to jam-types.asn in the jam-test-vectors repo
+
+U8 ::= INTEGER (0..255)
+U32 ::= INTEGER (0..4294967295)
+CoreIndex ::= INTEGER (0..65535)
+
+Hash ::= OCTET STRING (SIZE(32))
+HeaderHash ::= Hash
+StateRootHash ::= Hash
+
+-- Features is a u32 where the MSB (bit 31) is reserved for future extensions
+-- If MSB is 1, another u32 follows with more feature flags
+Features ::= U32
+
+-- Feature bit constants
+feature-block-ancestry INTEGER ::= 1      -- 2^0
+feature-simple-forking INTEGER ::= 2      -- 2^1
+feature-bundle-refinement INTEGER ::= 4   -- 2^2
+feature-extension INTEGER ::= 2147483648  -- 2^31 = 0x80000000
+
+Version ::= SEQUENCE {
+    major U8,
+    minor U8,
+    patch U8
+}
+
+PeerInfo ::= SEQUENCE {
+    -- Fuzzer protocol version
+    fuzz-version U8,
+    -- Application version
+    app-version  Version,
+    -- JAM protocol version
+    jam-version  Version,
+    -- Supported features
+    features Features,
+    -- Application name
+    name UTF8String
+}
+
+TrieKey ::= OCTET STRING (SIZE(31))
+
+KeyValue ::= SEQUENCE {
+    key     TrieKey,
+    value   OCTET STRING
+}
+
+State ::= SEQUENCE OF KeyValue
+
+ImportBlock ::= Block
+
+SetState ::= SEQUENCE {
+    header Header,
+    state State
+}
+
+GetState ::= HeaderHash
+
+StateRoot ::= StateRootHash
+
+--------------------------------------
+-- Refinement
+--------------------------------------
+
+RefineBundle ::= SEQUENCE {
+    bundle WorkPackageBundle,
+    segment-root-mappings  SegmentRootLookup
+}
```

Comment:
@davxy  Shouldn't RefineBundle include `CoreIndex`?

```
RefineBundle ::= SEQUENCE {
    core CoreIndex
    bundle WorkPackageBundle,
    segment-root-mappings  SegmentRootLookup
}
```

otherwise how would target set its coreIdx in report?

Reply:
Yes CoreIndex should be included